doc/openfortivpn.1.in

```diff
@@ -72,6 +73,10 @@ VPN account username.
 VPN account password in plain text.
 For a secure alternative, use pinentry or let openfortivpn prompt for the password.
 .TP
+\fB\-\-cookie=\fI<cookie>\fR
+A valid cookie (SVPNCOOKIE) to use in place of username and password.
+The value is read from standard input when \fI<cookie>\fR is -.
```

Comment:
Two options, `--cookie=<cookie>` and `--cookie-on-stdin`, like [OpenConnect](https://gitlab.com/openconnect/openconnect), might be easier for end-users. How does this sound?

In the same spirit, `--password=<password>` would be deprecated (not documented) and  `--passwd-on-stdin` coudl be added.

Reply:
This sounds good to me, I've updated the PR.